sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -524,10 +524,10 @@ class annotated_ref {
   public:
     annotated_ref(const annotated_ref&) = delete;
     operator T() const;
-    T operator=(const T &) const;
+    T operator=(T) const;
     T operator=(const annotated_ref&) const;
     // OP is: +=, -=, *=, /=, %=, <<=, >>=, &=, |=, ^=
-    T operatorOP(const T &) const;
+    T operatorOP(T) const;
     T operator++();
```

Comment:
Are you adding the missing `const` in a separate PR?

Reply:
I thought we will passs-by-value for all compound operators as well?

Reply:
I think @rolandschulz is asking to add `const` to the member function like this:

```
T operator++() const;
```

(Same for the other increment / decrement operators.)

Also, it looks like the table below is missing `const` on several of the member functions.  Please bring the table into alignment with the synopsis.

Reply:
I address this here https://github.com/intel/llvm/pull/11904